populate models with admin popularity/population metadata

sevko opened the pull request:
Extract population/popularity values from Quattro polys and inject them into inbound `Document`s coming through `.stream()` using `.setPopulation()` and `.setPopularity()`. Will facilitate results boosting for every record passing through the `admin-lookup`.

Implements #5 .
Fixes pelias/pelias#51

sevko committed:
Add population/popularity to records.

lib/master.js
	-Extract `qs_pop` and `popularity` population/popularity values
	from Quattroshapes shapefiles.
	-Add them to incoming `Document`s.

sevko committed:
Fix incorrect population retrieval.

lib/master.js
	-`locality` population was taken twice in a conditional
	selector, instead of one from `locality` and another from
	`local_admin`.

sevko committed:
Merge branch 'master' into admin-metadata

Merge now to avoid hemorrhage-inducing merge conflicts later on.

sevko committed:
Add population/popularity setter calls.

lib/master.js
	-Set `Document` popularity and population values via the
	respective setter methods.

sevko commented:
Needs to be tested in dev imports before getting merged.

hkrishna commented on lib/master.js:
```diff
@@ -134,18 +135,33 @@ function createLookup( workers ){
+      resps.neighborhood.popularity
+    ];
+    result.popularity = popularities.reduce( function ( acc, curr ){
+      return ( typeof curr === 'number' ) ?
```
Not sure if this check is sufficient.. Beware of Javascript demons! 

```javascript
  typeof NaN = 'number'
```
how about ```isNaN()``` ?

sevko replied:
Better safe than sorry, will add.

hkrishna commented on lib/master.js:
```diff
@@ -134,18 +135,33 @@ function createLookup( workers ){
+        acc + curr :
+        acc;
+    });
+    console.log( result );
```
is this for debugging purposes? should we delete this line before merging into master?

sevko replied:
Indeed, I accidentally committed this and then deleted locally (but never pushed).

sevko committed:
Add popularity/population to unit tests.

test/lookup_points.json
	-Add popularity/population values to the `npm run test-lookups`
	unit tests.

sevko committed:
Remove console.log, robustify number conditional.

lib/master.js
	-Remove a leftover, debugging `console.log()`.
	-Make a conditional that checks whether a value is a number more
	robust with `!isNaN()`.

hkrishna commented:
@sevko can you resolve conflicts and make this PR merge-able again? :)

hkrishna commented:
+1
